src/selectors/prevSibling.js

```diff
@@ -1,34 +1,50 @@
-import { getSortPreference, getThoughtsRanked, getThoughtsSorted, hasChild } from '../selectors'
-import { isFunction } from '../util'
+import { getContextsSortedAndRanked, getSortPreference, getThoughtsRanked, getThoughtsSorted, hasChild, isContextViewActive } from '../selectors'
+import { head, isFunction, perma } from '../util'
 
 /**
  * Gets a context's previous sibling with its rank.
  *
  * @param context   Can be a context or path.
  */
 export default (state, value, context, rank) => {
+
   const { showHiddenThoughts } = state
   const sortPreference = getSortPreference(state, context)
-  const siblings = (sortPreference === 'Alphabetical' ? getThoughtsSorted : getThoughtsRanked)(state, context)
+
+  const contextViewActive = perma(() => isContextViewActive(state, context))
+
+  /** Gets siblings of a context. */
+  const getContextSiblings = () => getContextsSortedAndRanked(state, head(context))
+
+  /** Gets siblings of thought. */
+  const getThoughtSiblings = () => (sortPreference === 'Alphabetical' ? getThoughtsSorted : getThoughtsRanked)(state, context)
+
+  const siblings = contextViewActive() ? getContextSiblings() : getThoughtSiblings()
   let prev// eslint-disable-line fp/no-let
 
+  /** Returns true when context is not hidden due to being a function or having a =hidden attribute. */
+  const isVisibleContext = contextRanked => showHiddenThoughts || (
```

Comment:
I believe what is passed is an unranked `context`.

Reply:
That's right, I'll update the name